<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGetWindowInfoClass</class>
 <widget class="QWidget" name="QtGetWindowInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGetWindowInfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>类</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>样式</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>句柄</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>标题</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>矩阵</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_windowHwnd"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_windowTitle"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_windowClass"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_windowStyle"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_windowRect"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtComposeResultCalculator.qrc"/>
 </resources>
 <connections/>
</ui>
